app/views/error.nunjucks

```diff
@@ -1,13 +1,13 @@
 {% extends 'layout.nunjucks' %}
 
 {% block header %}
-Ooops...an error has occured.
+A server error has occured.
 {% endblock %}
 
 {% block content %}
-<div onclick="(function myFunction() { var error = document.getElementById('error-message'); error.style.display = error.style.display == 'none' ? 'block' : 'none'; })()">
-Looks like one of the techies has pulled a sickie. Please try again or get in touch if it keeps happening.
-<div id="error-message" style="display:none;">
+<p>This error page should only appear in non-production envirnoments.</p>
```

Comment:
The aim here was to make this page substantially different from the Akamai one so, for example, if the Akamai config was incorrect in a particular environment it would be immediately obvious. I didn't want this page to leak through into production given that it displays error message.

I guess the answer is to remove the display of error messages so we have to rely on logging then if it does get displayed then it is not a problem.